### DISCUSSION TITLE
Welcome to TabPy Discussions!

### DISCUSSION THREAD

**Discussion by 0golovatyi** [tableau/TabPy#465] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by msacs09** [comment]
Did anyone achieve a successful integration of Tabpy with Anaconda Enterprise running on AWS and Tableau server running on on-perm ?

**Reply by 0golovatyi** [reply]
@msacs09 What issues do you see with that configuration? Please open an issue with repro steps. And yes, all should work :)
